packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -255,4 +261,10 @@ describe('Marketplace page', () => {
     const notInstalledText = queryByText(notInstalledCard, /copy install command/i);
     expect(notInstalledText).toBeVisible();
   });
+
+  it('Its showing the sort by menu and its options', async () => {
```

Comment:
I think we can be more ambitious with the tests there:

- check if the select appears ✅ 
- click on it and make sure the options are the ones we expect
- click on an option and make sure it writes the query params to the URL

Reply:
I think we could also add:
- click on an option and check the elements are sorted correctly